pre-processing/primod/ribamod/ribamod.py

```diff
@@ -8,9 +8,13 @@
 import pandas as pd
 import ribasim
 import tomli_w
-import xarray as xr
 from imod.mf6 import Drainage, GroundwaterFlowModel, Modflow6Simulation, River
 
+from primod.ribamod.exchange_creator import (
+    derive_active_coupling,
+    derive_passive_coupling,
```

Comment:
Should those two functions maybe be re-exported in `primod/ribamod/__init__.py`?

Reply:
Okay, then let's leave it as is